DependencyInjection/Configuration.php

```diff
@@ -81,7 +81,7 @@ public function getConfigTreeBuilder()
                             ->booleanNode('strict')->defaultTrue()->end()
                             ->booleanNode('escaping')->defaultTrue()->end()
                             ->booleanNode('line_breaks')->defaultTrue()->end()
-                            ->integerNode('max_newlines')->defaultValue(3)->end()
+                            ->scalarNode('max_newlines')->defaultValue(3)->end()
```

Comment:
@helios-ag It's an user depend decision.

What do it brings to drop this support ?
If we are limited with older Symfony version to improve the bundle then we should do it. With a dedicated branch named something like `Symfony-v2.1+` or better (this name idea is bad).

Related to #130